LeetCode 75/334 Increasing Triplet Subsequence.py

```diff
@@ -0,0 +1,19 @@
+# Given an integer array nums, return true if there exists a 
+# triple of indices (i, j, k) such that i < j < k and nums[i] < nums[j] < nums[k].
+#  If no such indices exists, return false.
+
+
+class Solution:
+    def increasingTriplet(self, nums: List[int]) -> bool:
+
+        first = second = float('inf')
```

Comment:
float('info')?

Reply:
This is useful for finding lowest values for something.
https://stackoverflow.com/questions/34264710/what-is-the-point-of-floatinf-in-python